### DISCUSSION TITLE
How to make a nicely aligned diagram

### DISCUSSION THREAD

**Discussion by lcnbr** [typst/typst#1831] [Q&A]
I am trying to make a nicely aligned diagram of implications in measure theory and I cannot satisfactorily align the different parts. Does anyone know how I should go about this in typst? Here is what I've got:

https://typst.app/project/royaWrohjA4Kb26UYEcWpA

**Comment by Enivex** [comment]
You may want to check out the commute package

**Reply by lcnbr** [reply]
I tried that but it isn't really made for this application as I need double lined arrows for implication
